src/cls/cephfs/cls_cephfs_client.h

```diff
@@ -1,5 +1,5 @@
-// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
-// vim: ts=8 sw=2 smarttab
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:nil -*-
+// vim: ts=8 sw=2 expandtab
```

Comment:
Why do some of these lines have `ft=cpp` but not others?

Reply:
Ack.  I dont' know if it's important but I note for due diligence.  Did you edit each file, or use a find/sed/awk loop?